static-site/src/pages/groups.jsx

```diff
@@ -0,0 +1,116 @@
+import UserDataWrapper from "../layouts/userDataWrapper";
+
+const nextstrainLogoPNG = require("../../static/logos/favicon.png");
+
+const title = "Scalable Sharing with Nextstrain Groups";
+const abstract = (<>
+  We want to enable research labs, public health entities
+  and others to share their datasets and narratives through Nextstrain with
+  complete control of their data and audience. Nextstrain Groups is more scalable
+  than community builds in both data storage and viewing permissions. Each group
+  manages its own AWS S3 Bucket to store datasets and narratives, allowing many
+  large datasets. Data of a public group are accessible to the general public via
+  nextstrain.org, while private group data are only visible to logged in users
+  with permissions to see the data. A single entity can manage both a public and
+  a private group in order to share data with different audiences.
+  <br/>
+  <br/>
+  For more details about Nextstrain Groups
+  , <a href="https://docs.nextstrain.org/en/latest/guides/share/nextstrain-groups.html">check out our documentation</a>.
+  <br/>
+  <br/>
+  Nextstrain Groups is still in the early stages and require a Nextstrain team
+  member to set up and add users.
+  Please <a href="mailto:hello@nextstrain.org">contact us at hello@nextstrain.org </a> and we’d be happy to set up a group for you.
+</>);
+const tableColumns = [
+  {
+    name: "Dataset",
+    value: (dataset) => dataset.filename.replace(/_/g, ' / ').replace('.json', ''),
+    url: (dataset) => dataset.url
+  },
+  {
+    name: "Source",
+    value: (dataset) => dataset.source
+  },
+  {
+    name: "Contributor",
+    value: (dataset) => dataset.contributor,
+    url: (dataset) => dataset.contributorUrl,
+    logo: (dataset) => dataset.contributor==="Nextstrain" ?
+      <img alt="nextstrain.org" className="logo" width="24px" src={nextstrainLogoPNG}/> :
+      <MdPerson/>
+  }
+];
+
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dataLoaded: false,
+      errorFetchingData: false,
+      datasetsUrl: "https://staging.nextstrain.org/james/tmp-all-datasets.json"
+    };
+  }
+  async componentDidMount() {
+    try {
+      const datasets = await fetchAndParseJSON(this.state.datasetsUrl);
+      this.setState({
+        datasets: datasets.filter((d) => d.source === "Public Group"),
+        dataLoaded: true
+      });
+    } catch (err) {
+      console.error("Error fetching / parsing data.", err.message);
+      this.setState({errorFetchingData: true});
+    }
+  }
+
+  render() {
+    return (
+      <GenericPage location={this.props.location}>
+        <splashStyles.H1>{title}</splashStyles.H1>
+        <SmallSpacer />
+
+        <FlexCenter>
+          <splashStyles.CenteredFocusParagraph>
+            {abstract}
+          </splashStyles.CenteredFocusParagraph>
+        </FlexCenter>
+
+        <HugeSpacer /><HugeSpacer />
+        <UserDataWrapper>
+          <UserGroups/>
```

Comment:
Once the cards have been refactored into their own component (see above), this section should be:

* Title (font size smaller than `<splashStyles.H1>`) of "Available groups"
* If logged in (i.e. `props.user` is set) byline should be "Click on any tile to view the different datasets and narratives available for that group. A padlock icon indicates a private group which you ({props.user}) have access to." 
* If not logged in, the byline should be "Click on any tile to view the different datasets and narratives available for that group. These groups are all public, to see private groups please log in"  (log in should be a link)
* render `<GroupCards>`

Reply:
Resolved by f345faf5667668e64b1173f0aee5c275c43e8e26